packages/components/src/VirtualizedList/ListTable/ListTable.scss

```diff
@@ -23,9 +23,8 @@ $tc-list-table-border: 1px solid darken($concrete, 5%);
 		}
 	}
 
-	.tc-list-item {
+	:global(.ReactVirtualized__Table__headerRow) {
```

Comment:
Please avoid using global scope in a CSS Module environment.

Reply:
Couldn't find a way to set a class on header row except this global selector. 
I tried `headerClassName` (https://github.com/bvaughn/react-virtualized/blob/master/docs/Table.md) but class is set on cells, not on full row. Resulting in wired box-shadow...